packages/core/components/EditMetadata/NewAnnotationPathway.tsx

```diff
@@ -137,14 +144,14 @@ export default function NewAnnotationPathway(props: NewAnnotationProps) {
                     <StackItem>
                         {step === EditStep.EDIT_FILES && (
                             <SecondaryButton
-                                title="Go back"
+                                title=""
```

Comment:
No title for the button?

Reply:
Is intentional. Was UX request since when the title is the same as the text on the button, the tooltip is distracting and doesn't add any info. Planned to make ticket to make titles optional for buttons